tests/scripts/components-basic-checks.sh

```diff
@@ -0,0 +1,15 @@
+# components-basic-checks.sh
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+# This file contains test components that are executed by all.sh
+
+################################################################
+#### Basic checks
+################################################################
+
+component_tf_psa_crypto_check_files () {
```

Comment:
The originating issue says we need 2 new components: `component_tf_psa_crypto_check_files` (added here) and `component_tf_psa_crypto_check_python_files` which I cannot find anywhere in the repo. Just to be sure, was it intentional to skip the 2nd?

Reply:
Ah, it was edited since I last saw it. Let me add a new component for `check_python_files`.